drivers/windows/overlay/ov_endpoint_windows.go

```diff
@@ -263,3 +269,14 @@ func (d *driver) EndpointOperInfo(nid, eid string) (map[string]interface{}, erro
 
 	return data, nil
 }
+
+func endpointRequest(method, path, request string) (*hcsshim.HNSEndpoint, error) {
+	if windowsBuild == 14393 {
```

Comment:
@pradipd is this a fixed version for WS 2016 server or will this change for every patch ?

Reply:
fixed version.  We (windows) should really have a constant for this.

Reply:
Thanks. We see it as a hcsshim constant `osversion.RS5`. Thanks.